README.md

```diff
@@ -60,25 +59,12 @@ It doesn't work for all anime, but the ones it can't find will print out alongsi
 
 ## Install
 
-### Arch
-
-Also consider ani-cli-git
-
-```sh
-yay -S ani-cli
-```
-
-### Other native packages
-
-[![Packaging status](https://repology.org/badge/vertical-allrepos/ani-cli.svg)](https://repology.org/project/ani-cli/versions)
-
```

Comment:
Why?

Reply:
if we shift to installation scripts, this would be pretty redundant and would also look bad for the project page if arent updated [as they are rn]